scripts/generate_lumos_terraform_resource_for_import.py

```diff
@@ -19,6 +19,19 @@ def __init__(self) -> None:
             "Content-Type": "application/json",
         }
 
+    def get_user(self, user_id: str) -> dict:
+        response = requests.get(
+            f"{self.LUMOS_API_ENDPOINT}/appstore/users/{user_id}",
+            headers=self._headers,
```

Comment:
Similar to an above comment, can we `get_user` with email as well?

Reply:
emails should be unique, right?